grpc/pb-marshalling.go

```diff
@@ -224,26 +218,20 @@ func pbToValidationResult(in *vapb.ValidationResult) ([]core.ValidationRecord, *
 	return recordAry, prob, nil
 }
 
-func performValidationReqToArgs(in *vapb.PerformValidationRequest) (domain string, challenge core.Challenge, authz core.Authorization, err error) {
-	if in == nil {
-		err = ErrMissingParameters
-		return
-	}
-	if in.Domain == nil {
-		err = ErrMissingParameters
-		return
+func performValidationReqToArgs(in *vapb.PerformValidationRequest) (string, core.Challenge, core.Authorization, error) {
+	if in == nil || in.Domain == nil {
```

Comment:
In other cases, like this one, I've left a check in for `.Domain` because as far as I can tell, there's never a situation where the domain should be the empty string, and therefore this check can be safely switched to `== ""` when the protos are updated.